http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -460,6 +465,10 @@ class LeoPubsubMessageSubscriber[F[_]: Timer: ContextShift](
       )
     } yield ()
 
+  def handleCreateAppMessage(msg: CreateAppMessage): F[Unit] = F.unit
```

Comment:
add TODO or use `NotImplemented` exception

Reply:
ill be implementing this in the ticket I have now, so it won't stay like this for long